src/pages/Checkout-first-step.page.js

```diff
@@ -0,0 +1,36 @@
+import { BaseSwagLabPage } from './BaseSwagLab.page';
+import casual from 'casual';
+
+export class CheckoutFirstStepPage extends BaseSwagLabPage {
+    url = '/checkout-step-one.html';
+    
+
+    get customerName() {return this.page.locator ('[id="first-name"]')};
+
+    get customerSurname() { return this.page.locator('[id="last-name"]')};
+
+    get customerZipcode() { return this.page.locator('[id="postal-code"]')};   
+    
+    get checkoutButton () {return this.page.locator ('[id="checkout"]')};  
+    
+    get continueButton () {return this.page.locator ('[id="continue"]')};  
+
+     async fillUserData () {
```

Comment:
Its better to have this function reusable. Lets provide user data as a parameter here.